### DISCUSSION TITLE
Is it possible to remap item changing keys on mini.starter?

### DISCUSSION THREAD

**Discussion by kvietcong** [nvim-mini/mini.nvim#77] [Q&A] [answered]
Instead of up/down/<m-j>/<m-k> to move up and down, could I add <c-n>/<c-p>?

**Comment by echasnovski** [comment, marked as answer]
I actually thought about adding `<C-n>` and `<C-p>` in default mappings, because muscle memory slipped several times into using them. Thanks for reminder!

But to answer your question: yes, of course. You can create an autogroup similar to [this one](https://github.com/echasnovski/mini.nvim/issues/70#issuecomment-1116459329) but with your own keys.

**Reply by kvietcong** [reply]
Thank you so much for the reply and for the quick feature implementation into main. Appreciate it a lot!

**Comment by echasnovski** [comment]
You should be able to move with `<C-n>` and `<C-p>` by default now in latest `main` branch.
